ignite-generator/listview/templates/listview.js.template

```diff
@@ -84,14 +89,50 @@ class <%= name %> extends React.Component {
     return this.state.dataSource.getRowCount() === 0
   }
 
+  // Magical helper function that can scroll your ListView to the bottom
+  scrollToBottom(animated = true) {
+    if (this.listHeight && this.footerY && this.footerY > this.listHeight) {
+      // Calculates the y scroll position inside the ListView
+      const scrollTo = this.footerY - this.listHeight
+
+      // Scroll that sucker!
+      this.refs.listView.scrollTo({
+        y: scrollTo,
+        animated: animated,
+      })
+    }
+  }
+
+  // Save the list's height when it renders
+  onLayout = (event) => {
+    const layout = event.nativeEvent.layout
+    this.listHeight = layout.height
+  }
+
+  // Render a fake footer for determining the height of the ListView
```

Comment:
Can you change this to 'render a footer.  Keep `onFooterLayout` if you decide to fill out this section'